src/callbacks_step/analysis_dgmulti.jl

```diff
@@ -135,6 +135,43 @@ function analyze(::Val{:linf_divb}, du, u, t,
   return linf_divB
 end
 
+function integrate(func::typeof(enstrophy), u,
+                   mesh::DGMultiMesh,
+                   equations, equations_parabolic::CompressibleNavierStokesDiffusion3D,
```

Comment:
Same comment as for the `TreeMesh{3}` version.

Reply:
I think however we decide to dispatch the `TreeMesh{3}` version its analog will work for `DGMulti`